targets/ath79-tiny

```diff
@@ -0,0 +1,22 @@
+config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
+
+no_opkg()
+
+packages {
+	'-uboot-envtools',
+	'-kmod-usb-core',
+	'-kmod-usb-ohci',
+	'-kmod-usb2',
+	'-kmod-usb-ledtrig-usbport',
+	'-kmod-usb-chipidea2',
+}
+
+defaults {
+	deprecated = true, -- 4/32
+}
+
+
+-- TP-Link
+
+device('tp-link-tl-mr3020-v1', 'tplink_tl-mr3020-v1')
+device('tp-link-tl-wr-740n-v1', 'tplink_tl-wr740n-v1')
```

Comment:
If we add these models in ath79 targets, shouldn't we drop them in the corresponding ar71xx target at the same time?

Reply:
This decision should be based on whether or not the migration was tested or not.

The current selection of devices in the new targets is only based on what I've had lying around.

Reply:
So they were tested?

Reply:
good question. i think we should only enable fully tested devices until we have decided on something else.

Reply:
I agree on this topic. As this PR is only an RFC (and someone else might want to try out the ath79 target), I've added targets such as lantiq or ath79. If we come close to migrating over to 19.07, we should definitely not carry these over to the master branch without sufficient testing.